packages/editor/playground/src/utils/update-url.ts

```diff
@@ -0,0 +1,15 @@
+interface URLOptions {
+    contentId?: string
+    enableDevTools?: boolean
+}
+export default function updateURLParams(o: URLOptions): void {
```

Comment:
```suggestion
export default function updateURLParams(options: URLOptions): void {
```

According to my observation, unlike Go community, JS community usually don't use a single letter for variable name.